llvm/include/llvm/IR/IntrinsicsAArch64.td

```diff
@@ -3373,7 +3373,7 @@ let TargetPrefix = "aarch64" in {
   // Multi-vector min/max
   //
 
-  foreach ty = ["f", "s", "u"] in {
+  foreach ty = ["bf", "f", "s", "u"] in {
```

Comment:
You could just omit that part. Then the `bfloat` intrinsics would use `fmin`/`fmax`/etc in the names without ambiguity, since they are polymorphic.